images/linux/Ubuntu2004-README.md

```diff
@@ -4,31 +4,31 @@
 | [Clang/LLVM 10 will be set as a default one and Clang/LLVM 6 will be deprecated  for Ubuntu 20.04 on September, 23](https://github.com/actions/virtual-environments/issues/1536) |
 ***
 # Ubuntu 20.04.1 LTS
-- Image Version: 20200920.1
+- Image Version: 20201004.1
 
 ## Installed Software
 ### Language and Runtime
-- GNU C++ 7.5.0, 8.4.0, 9.3.0
+- GNU C++ 7.5.0, 8.4.0, 9.3.0, 10.0.1
 - GNU Fortran 8.4.0, 9.3.0
-- Clang 10.0.1, 8.0.1, 9.0.1
-- Erlang 11.0.3
+- Clang 8.0.1, 9.0.1, 10.0.0
+- Erlang 
 - Mono 6.12.0.90
 - Node 12.18.4
-- Python 2.7.18rc1
+- Python 3.8.2
 - Python3 3.8.2
 - PowerShell 7.0.3
 - Ruby 2.7.0p0
 - Swift 5.3
-- Julia 1.5.1
+- Julia 1.5.2
 
 ### Package Management
-- Homebrew 2.5.1
+- Homebrew 2.5.2
 - Gem 3.1.2
 - Miniconda 4.8.3
-- Helm 3.3.3
+- Helm 3.3.4
 - Npm 6.14.8
 - Yarn 1.22.5
-- Pip 20.2.3
+- Pip 20.0.2
```

Comment:
Looks like we need to update the version back

Reply:
https://github.com/actions/virtual-environments/pull/1705
now pip is the link to pip3, because we have same versions for pip and pip3